doc/user/content/manage/user-management/_index.md

```diff
@@ -0,0 +1,120 @@
+
+### Role attributes
+
+Role attributes are actions available to any role you create. Attributes are
+independent of any other object in Materialize and apply to the entire
+organization. You can edit these actions when you create the role:
+
+| Name            | Description                                                                     |
+|-----------------|---------------------------------------------------------------------------------|
+| `CREATEDB`      | Can create a database.                                                          |
+| `CREATEROLE`    | Can create, alter, drop, grant membership to, and revoke membership from roles. |
+| `INHERIT`       | **Read-only.** Can inherit the privileges of roles that it is a member of. On by default.      |
+| `CREATECLUSTER` | Can create a cluster.                                                           |
+
+
+### Privileges
+
+Privileges are the actions or operations a role is allowed to perform on a
+specific object. After you create a role, you can grant it the following
+object-specific privileges in Materialize:
+
+| Privilege | Description                                                | `psql` |
+| --------- | ---------------------------------------------------------- | ------ |
+| `SELECT`  | Allows selecting rows from an object.                      | `r`    |
+| `INSERT`  | Allows inserting into an object.                           | `a`    |
+| `UPDATE`  | Allows updating an object (requires `SELECT`).             | `w`    |
+| `DELETE`  | Allows deleting from an object (requires `SELECT`).        | `d`    |
+| `CREATE`  | Allows creating a new object within another object.        | `C`    |
+| `USAGE`   | Allows using an object or looking up members of an object. | `U`    |
+
+Note that the system catalog uses the abbreviation of the privilege name.
+
+Objects in Materialize have different levels of privileges available to them.
+Materialize supports the following object type privileges:
+
+| Object Type          | Privileges                          |
+|----------------------|-------------------------------------|
+| `DATABASE`           | `USAGE` `CREATE`                    |
+| `SCHEMA`             | `USAGE` `CREATE`                    |
+| `TABLE`              | `INSERT` `SELECT` `UPDATE` `DELETE` |
+| `VIEW`               | `SELECT`                            |
+| `MATERIALIZED  VIEW` | `SELECT`                            |
+| `TYPE`               | `USAGE`                             |
+| `SOURCE`             | `SELECT`                            |
+| `CONNECTION`         | `USAGE`                             |
+| `SECRET`             | `USAGE`                             |
+| `CLUSTER`            | `USAGE` `CREATE`                    |
+
+
+### Inheritance
+
+Inheritance in RBAC allows you to create roles that inherit permissions
+from other roles. Inheriting permissions allows you to minimize the number of
+roles you have to manage.
+
+## Next steps
```

Comment:
Have you considered adding 1 or 2 simple examples to clarify how things fit together? The description is quite generic (which isn't a bad thing). It would have helped me to have short example that explains the concepts with a concrete example. Something like there is this role with these privileges on these views, so a user can access this view but not that view. Similar to the guide, but much more condensed and not meant to replicate.

Reply:
Hmm, I understand that having some examples may be helpful to explain these concepts, but I'm not sure if having examples after each section or a sample scenario would be more useful. Like a short blurb of "X role has x permissions and if they try to access y, they get this error" at the end? Could you highlight some sections that were too hand-wavey? I can see an explanation helping with the Inheritance section, but there are probably more so please let me know.

Reply:
We'll be adding some more guides for use cases as features come out so we'll have some more short examples to add in the future.